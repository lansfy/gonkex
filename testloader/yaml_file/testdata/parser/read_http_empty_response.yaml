- name: simple request test without response
  method: GET
  path: /some/path
  request: "requestbody"
