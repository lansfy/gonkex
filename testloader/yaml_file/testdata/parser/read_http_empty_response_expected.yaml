- GetName: simple request test without response
  GetMethod: GET
  Path: /some/path
  GetRequest: requestbody
  GetFileName: testdata/parser/read_http_empty_response.yaml
  FirstTestInFile: true
  LastTestInFile: true
